unicode_smilies/unicode_smilies.php

```diff
@@ -45,555 +45,552 @@ function unicode_smilies_smilies(&$a,&$b) {
-//	Smilies::add($b, ':carrot', '&#x1F955;'); // Bad client support
-	Smilies::add($b, ':popcorn', '&#x1F37F;');
-	Smilies::add($b, ':tomato', '&#x1F345;');
-	Smilies::add($b, ':banana', '&#x1F34C;');
-	Smilies::add($b, ':cooking', '&#x1F373;');
-	Smilies::add($b, ':fryegg', '&#x1F373;');
-	Smilies::add($b, ':birthday cake', '&#x1F382;');
+	Smilies::add($b, ':apple:', '&#x1F34E;');
+//	Smilies::add($b, ':broccoli:', '&#x;');
+	Smilies::add($b, ':cake:', '&#x1F370;');
+//	Smilies::add($b, ':carrot:', '&#x1F955;'); // Bad client support
+	Smilies::add($b, ':popcorn:', '&#x1F37F;');
+	Smilies::add($b, ':tomato:', '&#x1F345;');
+	Smilies::add($b, ':banana:', '&#x1F34C;');
+	Smilies::add($b, ':cooking:', '&#x1F373;');
+	Smilies::add($b, ':fryegg:', '&#x1F373;');
+	Smilies::add($b, ':birthday cake:', '&#x1F382;');
 
 // Happy smileys
 
-//	Smilies::add($b, ':cloud9', '&#x;');
-	Smilies::add($b, ':tearsofjoy', '&#x1F602;');
+//	Smilies::add($b, ':cloud9:', '&#x;');
+	Smilies::add($b, ':tearsofjoy:', '&#x1F602;');
 
 // Respect smileys
 
-	Smilies::add($b, ':bow', '&#x1F647;');
-//	Smilies::add($b, ':bravo', '&#x;');
-//	Smilies::add($b, ':hailking', '&#x;');
-//	Smilies::add($b, ':number1', '&#x;');
+	Smilies::add($b, ':bow:', '&#x1F647;');
+//	Smilies::add($b, ':bravo:', '&#x;');
+//	Smilies::add($b, ':hailking:', '&#x;');
+//	Smilies::add($b, ':number1:', '&#x;');
 
 // Laugh smileys
 
-//	Smilies::add($b, ':hahaha', '&#x;');
-//	Smilies::add($b, ':loltv', '&#x;');
-//	Smilies::add($b, ':rofl', '&#x1F923;'); // Bad client support
+//	Smilies::add($b, ':hahaha:', '&#x;');
+//	Smilies::add($b, ':loltv:', '&#x;');
+//	Smilies::add($b, ':rofl:', '&#x1F923;'); // Bad client support
 // Music smileys
 
-//	Smilies::add($b, ':drums', '&#x1F941;'); // Bad client support
-	Smilies::add($b, ':guitar', '&#x1F3B8;');
-	Smilies::add($b, ':trumpet', '&#x1F3BA;');
+//	Smilies::add($b, ':drums:', '&#x1F941;'); // Bad client support
+	Smilies::add($b, ':guitar:', '&#x1F3B8;');
+	Smilies::add($b, ':trumpet:', '&#x1F3BA;');
 
 // Smileys that used to be in core
 
-//	Smilies::add($b, ':headbang', '&#x;');
-//	Smilies::add($b, ':beard', '&#x1F9D4;'); // Bad client support
-//	Smilies::add($b, ':whitebeard', '&#x;');
-//	Smilies::add($b, ':shaka', '&#x;');
-//	Smilies::add($b, ':\\.../', '&#x;');
-//	Smilies::add($b, ':\\ooo/', '&#x;');
-//	Smilies::add($b, ':headdesk', '&#x;');
+//	Smilies::add($b, ':headbang:', '&#x;');
+//	Smilies::add($b, ':beard:', '&#x1F9D4;'); // Bad client support
+//	Smilies::add($b, ':whitebeard:', '&#x;');
+//	Smilies::add($b, ':shaka:', '&#x;');
+//	Smilies::add($b, ':\\.../:', '&#x;');
+//	Smilies::add($b, ':\\ooo/:', '&#x;');
+//	Smilies::add($b, ':headdesk:', '&#x;');
 
 // These two are still in core, so oldcore isn't strictly right, but we don't want too many directories
 
-//	Smilies::add($b, ':-d', '&#x;');
-	Smilies::add($b, ':-o', '&#x1F62E;');
+//	Smilies::add($b, ':-d:', '&#x;');
+	Smilies::add($b, ':-o:', '&#x1F62E;');
```

Comment:
This doesn't look right either, both of these are meant to represent the actual smiley. Please either remove the trailing colon or disable the replacement altogether.

Reply:
changed